projects/Mallard/ios/Podfile.lock

```diff
@@ -318,4 +318,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: c499c44aed47275a8e1f036511ecda7d7450e47a
 
-COCOAPODS: 1.6.1
+COCOAPODS: 1.7.5
```

Comment:
Do we want to do this, updating the cocoapods version will require changes to the build agent as we currently just use a local cocoapods install?

Reply:
You are right, we dont want to do this. I need to downgrade my machine.